src/player.ts

```diff
@@ -28,6 +28,81 @@ export interface PlayerConfig extends NativeInstanceConfig {
    * ```
    */
   licenseKey?: string;
+  /**
+   * Configuration options for player's default playback behavior.
+   */
+  playbackConfig?: PlaybackConfig;
+}
+
+/**
+ * Configuration options for player's default playback behavior.
+ */
+export interface PlaybackConfig {
+  /**
+   * Whether the player starts playing automatically after loading a source or not. Default is `false`.
+   * @example
+   * ```
+   * const player = new Player({
+   *   playbackConfig: {
+   *     isAutoplayEnabled: true,
+   *   },
+   * });
+   * ```
+   */
+  isAutoplayEnabled?: boolean;
+  /**
+   * Whether the sound is muted on startup or not. Default value is `false`.
+   * @example
+   * ```
+   * const player = new Player({
+   *   playbackConfig: {
+   *     isMuted: true,
+   *   },
+   * });
+   * ```
+   */
+  isMuted?: boolean;
+  /**
+   * Whether time shift / DVR for live streams is enabled or not. Default is `true`.
+   *  @example
+   * ```
+   * const player = new Player({
+   *   playbackConfig: {
+   *     isTimeShiftEnabled: false,
+   *   },
+   * });
+   * ```
+   */
+  isTimeShiftEnabled?: boolean;
+  /**
+   * Optionally isBackgroundPlaybackEnabled: Specifies if isBackgroundPlaybackEnabled should be enabled.
+   *
+   * This param only work on iOS.
```

Comment:
Please always use the native SDKs documentation as they provide detailed explanation for given configurations which might be crucial for their usage.

```suggestion
   * Whether background playback is enabled or not.
   * Default is `false`.
   * 
   * When set to `true`, playback is not automatically paused
   * anymore when the app moves to the background.
   * When set to `true`, also make sure to properly configure your app to allow
   * background playback.
   *
   * On tvOS, background playback is only supported for audio-only content.
```